packages/rich-text/src/component/use-input-and-selection.js

```diff
@@ -287,7 +287,6 @@ export function useInputAndSelection( props ) {
 					end: index,
 					activeFormats: EMPTY_ACTIVE_FORMATS,
 				};
-				onSelectionChange( index, index );
```

Comment:
So this should be a small (maybe not noticeable) perf improvement, right?

Reply:
It's a huge improvement for `focus`:

```
│        focus         │                '0.27 ms'                 │  '32.6 ms'   │
│       minFocus       │                '0.24 ms'                 │  '0.41 ms'   │
│       maxFocus       │                '0.36 ms'                 │  '48.82 ms'  │
```

Reply:
Maybe we should update the metric though. It feels now that we should be computing "focus + selection change". It seems the time has moved to another event that we don't take into consideration in the metric itself?